libcrux-ml-kem/src/mlkem1024.rs

```diff
@@ -45,9 +50,14 @@ pub type MlKem1024PublicKey = MlKemPublicKey<CPA_PKE_PUBLIC_KEY_SIZE_1024>;
 /// Am ML-KEM 1024 Key pair
 pub type MlKem1024KeyPair = MlKemKeyPair<SECRET_KEY_SIZE_1024, CPA_PKE_PUBLIC_KEY_SIZE_1024>;
 
+/// An Unpacked ML-KEM 1024 Public key
+pub type MlKem1024PublicKeyUnpacked<Vector> = MlKemPublicKeyUnpacked<RANK_1024, Vector>;
+/// Am Unpacked ML-KEM 1024 Key pair
```

Comment:
These type aliases need the trait bounds for the C extraction to work.

Reply:
Done